packages/flutter_tools/lib/src/commands/build_windows.dart

```diff
@@ -19,10 +19,17 @@ import 'build.dart';
 
 /// A command to build a windows desktop target through a build shell script.
 class BuildWindowsCommand extends BuildSubCommand {
-  BuildWindowsCommand() {
+  BuildWindowsCommand({ bool verboseHelp = false }) {
     addTreeShakeIconsFlag();
-    addBuildModeFlags();
     usesTargetOption();
+    addBuildModeFlags(verboseHelp: verboseHelp);
+    usesPubOption();
+    addSplitDebugInfoOption();
+    addDartObfuscationOption();
+    usesDartDefineOption();
+    usesExtraFrontendOptions();
+    addEnableExperimentation(hide: !verboseHelp);
+    usesTrackWidgetCreation(verboseHelp: verboseHelp);
```

Comment:
Should all the desktop builds have all of these? Would it make sense to make a new class between BuildSubCommand and Build<desktop platform>Command with all of this so we're not maintaining multiple copies?

Reply:
Going forward, almost all the basic dart options should be supported across all builds, so it wouldn't need to be desktop specific either. Leaving this for another change